### ISSUE TITLE
Disable bounceable animation

### ISSUE THREAD

**Issue by enestasdemir** [WrathChaos/react-native-multiple-select#1]
Hi,

How can I disable bounceable animation on list toggle?

Thanks.

**Comment by kuraydev** [comment]
Hello @enestasdemir do you mean checkbox bounce effect right? If it is, right now you cannot disable it, but today I will update the react-native-bouncy-checkbox and you will disable the bounce effect. I will inform you when it is available.

**Comment by enestasdemir** [comment]
> Hello @enestasdemir do you mean checkbox bounce effect right? If it is, right now you cannot disable it, but today I will update the react-native-bouncy-checkbox and you will disable the bounce effect. I will inform you when it is available.

No, not for the checkbox. There is an animation that plays while the list is opening, can we disable it?

**Comment by kuraydev** [comment]
Oh I got it, right now it is not possible. What kind of opening animation do you need? like opacity?

**Comment by enestasdemir** [comment]
@WrathChaos I prefer a "slide down" style animation that we often see in accordion menus.

Also thanks for this component. It is very nice 👍

**Comment by kuraydev** [comment]
I guess it will take time, but of course, PR's are welcome. I will try to add this animation ASAP

**Comment by MrRevengerX** [comment]
Any progress on this? It is nice we can disable checkbox bounce animations @WrathChaos
